src/AddItems.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const updateGroceryItem = (event) => {
+    setGroceryItem(event.target.value);
+  };
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: document.querySelectorAll('input[name=frequency]:checked')[0]
+        .value,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
```

Comment:
After the form is submitted, it would be a good idea to reset at least the text input so that the user is aware the grocery item was added. 

```suggestion
    db.collection(userToken).add({ formData });
    setGroceryItem('')
```

Reply:
Oh good call! I forgot that we were leaving the text input on submit.